src/views/CurrentImageView.js

```diff
@@ -25,6 +25,6 @@ export function CurrentImage(props) {
 
 CurrentImage.propTypes = {
   selectImage: PropTypes.func,
-  currentImage: PropTypes.string,
+  currentImage: PropTypes.object,
```

Comment:
for objects, we can make a stronger proptype check if we use `PropTypes.shape`, and then specify what the keys are of that object. ie:

```js
currentImage: PropTypes.shape({
  _id: PropTypes.string,
  ...
})
```

also if we find the same objects being used a lot in components (like data models), we can even define them in an external file and import them in a bunch of places. not sure if this is particularly relevant to your app yet, but here's an example anyway:

```js
export const ImageModel = PropTypes. shape({
  _id: PropTypes.string;
  ...
});

...

import { ImageModel } from './someGlobalTypesFile';

SomeComponent.propTypes = {
  currentImage: ImageModel
}
```